test/onnx/test_fx_op_consistency.py

```diff
@@ -188,6 +188,9 @@
         "addmm", dtypes=onnx_test_common.BOOL_TYPES,
         reason=onnx_test_common.reason_onnx_does_not_support("Addmm")
     ),
+    xfail(
```

Comment:
```suggestion
    xfail_torchlib_forward_compatibility(
```

Could you rebase on a commit after https://github.com/pytorch/pytorch/pull/110536 and use `xfail_torchlib_forward_compatibility`? Thanks!